articles/azure/azs-how-create-vm-from-snapshot-portal.md

```diff
@@ -0,0 +1,154 @@
+toc_fullpath: Users/How To/Create a Virtual Machine from disk snapshot/azs-how-create-vm-from-snapshot-portal.md
+toc_mdlink: azs-how-create-vm-from-snapshot-portal.md
+---
+
+# How to create a virtual machine from a disk snapshot using the UKCloud Azure Stack Hub portal
+
+## Overview
+
+This article shows you how to use the UKCloud Azure Stack Hub portal to:
+
+* Take a snapshot of a managed disk
+
+* Create a new managed disk from the snapshot
+
+* Create a virtual machine from the new managed disk
+
+### Intended audience
+
+To complete the steps in this article, you must have appropriate access to a subscription in the Azure Stack Hub portal.
+
+## Creating a snapshot from a disk
+
+1. Log in to the Azure Stack Hub portal.
+
+    For more detailed instructions, see the [*Getting Started Guide for UKCloud for Microsoft Azure*](azs-gs.md).
+
+2. Click **All services** in the favourites panel, then select **Disks** under the *Compute* section.
+
+    ![All services - disks](images/azs-browser-allservices-disks.png)
+
+3. From the list, select the disk that you would like to create a snapshot from.
+
+    ![Disks blade](images/azs-browser-disks-list.png)
+
+    > [!NOTE]
+    > Snapshots can only be created from managed disks.
+
+4. In the new blade for the selected disk, click **Create snapshot**.
+
+    ![Create snapshot from disk - New Blade](images/azs-create-snapshot-disk.png)
+
+5. In the **Create snapshot** blade, enter the following information:
+
+    - **Name** - The name of the snapshot
+
+    - **Subscription** - This will be your UKCloud for Microsoft Azure subscription
+
+    - **Resource group** - Select an existing resource group, or create a new one by typing a name for your new resource group
+
+    - **Location** - This will be the location of the Azure Stack Hub
+
+    - **Account type** - Choose either Standard (HDD) or Premium (SSD) storage
+
+    ![Create snapshot from disk - Populate & Create](images/azs-browser-create-snapshot.png)
+
+6. When you're done, click the **Create** button to create the snapshot.
+
+7. You can monitor the progress of the snapshot creation by clicking the **Notifications** icon.
+
+    ![Notification showing snapshot creation in progress](images/azs-browser-create-snapshot-progress.png)
+
+## Creating a new managed disk from a snapshot
+
+1. Click **All services** in the favourites panel, then select **Disks** under the *Compute* section.
+
+    ![All services - disks](images/azs-browser-allservices-disks.png)
+
+2. On the **Disks** blade, click **Add**.
+
+    ![Disks blade - Add](images/azs-browser-disks-list-add.png)
+
+3. In the **Create managed disk** blade, enter the following information:
+
+    - **Name** - The name of the disk
+
+    - **Subscription** - This will be your UKCloud for Microsoft Azure subscription
+
+    - **Resource group** - Select an existing resource group, or create a new one by typing a name for your new resource group
```

Comment:
Select an existing resource group, or create a new one by clicking the **Create new** link and then typing a name for your new resource group in the pop out window.

Reply:
Thanks for highlighting this.